openbb_terminal/stocks/fundamental_analysis/av_view.py

```diff
@@ -251,6 +270,7 @@ def display_fraud(ticker: str, export: str = "", help_text: bool = False):
     if help_text:
         console.print(help_message)
     export_data(export, os.path.dirname(os.path.abspath(__file__)), "dupont", df)
+    return
```

Comment:
no need for this

Reply:
I was just doing it to match the other return so pylint would not throw an error.